charts/gardener-dashboard/charts/runtime/templates/dashboard/configmap.yaml

```
@@ -52,9 +52,13 @@ data:
       {{- end }}
       org: {{ .Values.global.dashboard.gitHub.org }}
       repository: {{ .Values.global.dashboard.gitHub.repository }}
-      {{- if .Values.global.dashboard.gitHub.pollIntervalSeconds }}
-      pollIntervalSeconds: {{ .Values.global.dashboard.gitHub.pollIntervalSeconds }}
-      {{- end }}
+      synchronization:
+        {{- if .Values.global.gitHub.webhookSecret }}
+        throttleSeconds: {{ required ".Values.global.gitHub.synchronization.throttleSeconds is required" .Values.global.gitHub.synchronization.throttleSeconds }}
+        intervalSeconds: {{ .Values.global.gitHub.synchronization.intervalSeconds | default 0 }}
+        {{- else }}
+        intervalSeconds: {{ required ".Values.global.gitHub.synchronization.intervalSeconds is required" .Values.global.gitHub.synchronization.intervalSeconds }}
+        {{- end }}
```

```suggestion
      synchronization:
        {{- if .Values.global.dashboard.gitHub.webhookSecret }}
        throttleSeconds: {{ required ".Values.global.dashboard.gitHub.synchronization.throttleSeconds is required" .Values.global.dashboard.gitHub.synchronization.throttleSeconds }}
        intervalSeconds: {{ .Values.global.gitHub.synchronization.intervalSeconds | default 0 }}
        {{- else }}
        intervalSeconds: {{ required ".Values.global.dashboard.gitHub.synchronization.intervalSeconds is required" .Values.global.dashboard.gitHub.synchronization.intervalSeconds }}
        {{- end }}
```
But this is not sufficient because we have no default for `.Values.global.dashboard.gitHub.synchronization`. It will be a breaking change.